### PR TITLE
Add support for Resolved issue state

### PR DESCRIPTION
**Pull request by rwestberg**
Hi all,

Please review this small change that adds support for setting issues to a Resolved state. Currently only supported by Jira, GitLab and GitHub will treat it as Closed.

Best regards,
Robin

## Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

## Approvers
 * Erik Helin ([ehelin](@edvbld) - **Reviewer**)

### CODE REVIEW

File: issuetracker/src/main/java/org/openjdk/skara/issuetracker/Issue.java [hunk 1]
```diff
@@ -104,6 +104,7 @@
 
     enum State {
         OPEN,
+        RESOLVED,
```
Comment (rwestberg) [inline issuetracker/src/main/java/org/openjdk/skara/issuetracker/Issue.java]: Another option would be to simply map CLOSED to the Jira Resolved state as that's probably what we want to always use..

Comment (rwestberg) [reply issuetracker/src/main/java/org/openjdk/skara/issuetracker/Issue.java]: Jira also supports renaming its states, so it's possible that we'll need to switch to setting State as a string to be interpreted as the particular Issue implementation sees fit.

Comment (edvbld) [reply issuetracker/src/main/java/org/openjdk/skara/issuetracker/Issue.java]: I think using `RESOLVED` makes sense for now :+1:

**Comment by rwestberg** [comment]
Thanks for reviewing!

**Pull request closed without merging**